modules/otel-config-send-metrics-monitoring-stack.adoc

```diff
@@ -1,12 +1,14 @@
-:_mod-docs-content-type: ASSEMBLY
-[id="otel-configuration-for-sending-metrics-to-the-monitoring-stack"]
+// Module included in the following assemblies:
+//
+// * observability/otel/otel-configuring-metrics-for-monitoring-stack.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="configuration-for-sending-metrics-to-the-monitoring-stack_{context}"]
 = Configuration for sending metrics to the monitoring stack
-include::_attributes/common-attributes.adoc[]
-:context: otel-configuration-for-sending-metrics-to-the-monitoring-stack
 
-The OpenTelemetry Collector custom resource (CR) can be configured to create a Prometheus `ServiceMonitor` CR for scraping the Collector's pipeline metrics and the enabled Prometheus exporters.
+A configured OpenTelemetry Collector custom resource (CR) can create a Prometheus `ServiceMonitor` CR for scraping the Collector's pipeline metrics and the enabled Prometheus exporters.
 
-.Example of the OpenTelemetry Collector custom resource with the Prometheus exporter
+.Example of the OpenTelemetry Collector CR with the Prometheus exporter
 [source,yaml]
```

Comment:
I know this is pre-existing content, but this yaml example is missing required fields per the OpenShift doc guidelines (https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#required-fields). You need at least `apiVersion` and `kind`.

Reply:
Thank you. Good to know. We'll fix it.

@pavolloffay, :slightly_smiling_face: would you mind to provide the `apiVersion` and `kind` for this block in reply to this comment?

Reply:
@max-cx our product has only a single API version at the moment. 

```yaml
apiVersion: opentelemetry.io/v1alpha1
kind: OpenTelemetryCollector
```

Reply:
ACK. Thank you.